src/Compilers/VisualBasic/Portable/Parser/ParserExtensions.vb

```diff
@@ -16,6 +16,81 @@ Imports InternalSyntaxFactory = Microsoft.CodeAnalysis.VisualBasic.Syntax.Intern
 Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
     Friend Module ParserExtensions
 
+        <Extension>
+        Function IsIn(kind As SyntaxKind,
+                      kind0 As SyntaxKind, kind1 As SyntaxKind, kind2 As SyntaxKind, kind3 As SyntaxKind, kind4 As SyntaxKind,
+                      kind5 As SyntaxKind, kind6 As SyntaxKind, kind7 As SyntaxKind, kind8 As SyntaxKind, kind9 As SyntaxKind) As Boolean
+            Return (kind = kind0) Or (kind = kind1) Or (kind = kind2) Or (kind = kind3) Or (kind = kind4) Or
```

Comment:
I think `OrElse` should perform better here.

Reply:
Not in this case, as it compile to just `or` without any branches. "OrElse" would add conditional branch after each comparison.